kube/minikube/deployments/timeout-closing-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kernelci-pipeline-timeout-closing
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kernelci-pipeline-timeout-closing
+  template:
+    metadata:
+      labels:
+        app: kernelci-pipeline-timeout-closing
+    spec:
+      containers:
+        - name: kernelci-pipeline-timeout-closing
+          image: kernelci/staging-kernelci
+          env:
+            - name: KCI_API_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: kernelci-pipeline-secrets
+                  key: kci-api-token
+            - name: SETTINGS
+              valueFrom:
+                configMapKeyRef:
+                  name: kernelci-pipeline-config
+                  key: settings
+          command:
+            - "/usr/bin/env"
+            - "python3"
+            - "/home/kernelci/pipeline/timeout.py"
+            - "--settings=$(SETTINGS)"
+            - "run"
+            - "--mode=closing"
+            - "--api-config=k8s-host"
```

Comment:
Is this not already in the TOML settings file?

Reply:
In the case of Minikube, TOML file is being used from `/home/kernelci/config/kernelci.toml` location.
That's why `api-config` needs to be passed here.
I think I need to add a different TOML file for Minikube as well like we have one for AKS.

Reply:
Yes, I think it's expected to have a different settings file for every use-case.  We may also need to use YAML config overlays, but that's a bit different.